libs/pika/async_mpi/tests/unit/CMakeLists.txt

```diff
@@ -4,19 +4,20 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-set(tests algorithm_transform_mpi mpi_ring_async_sender_receiver
-          mpi_async_storage
+set(tests # algorithm_transform_mpi
+    mpi_ring_async_sender_receiver
+    # mpi_async_storage
```

Comment:
Uncomment others before merging.

Reply:
The other tests are currently broken.

Reply:
By the way, I take your comment as an explanation for why they're uncommented, but I assume they will be updated in this PR?

Reply:
https://github.com/pika-org/pika/issues/612